lib/ramble/docs/dev_guides/application_dev_guide.rst

```diff
@@ -265,7 +265,6 @@ directives that are intended to be package manager specific. As an example,
 there are directives for Spack defined by:
 
 * :meth:`ramble.language.shared_language.software_spec`
-* :meth:`ramble.language.shared_language.default_compiler`
```

Comment:
I think instead of just removing default compiler, we should replace it with `define_compiler`

Reply:
Good catch, fixed.